recipes-kernel/linux/files/iot2050_defconfig_extra.cfg

```diff
@@ -200,3 +200,5 @@ CONFIG_DM_CRYPT=y
 
 CONFIG_TEE_STMM_EFI=m
 CONFIG_EFIVAR_FS=m
+
+CONFIG_TCG_FTPM_TEE=y
```

Comment:
I wonder if we should promote all the TEE and EFI related switches to `iot2050_defconfig_base` - they all refer to the standard firmware, at least soon.